### PR TITLE
[15.0][MIG] sale_stock_secondary_unit: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36545

ping @chienandalu @CarlosRoca13

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Ready for review

**Comment by sergio-teruel** [comment]
ping @CarlosRoca13 @chienandalu

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by sergio-teruel** [comment]
ping @chienandalu

File: sale_stock_secondary_unit/tests/test_sale_stock_secondary_unit.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestSaleStockOrderSecondaryUnit(SavepointCase):
```
Comment (CRogos) [inline sale_stock_secondary_unit/tests/test_sale_stock_secondary_unit.py]: There is a warning in logs. Can you replace it with TransactionCase?

**Review by CRogos — CHANGES_REQUESTED** [review]
Code review LGTM.

File: sale_stock_secondary_unit/tests/test_sale_stock_secondary_unit.py (hunk 1 above)
Comment (sergio-teruel) [reply sale_stock_secondary_unit/tests/test_sale_stock_secondary_unit.py]: Changes done!!

**Review by CRogos — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Pull request merged**